paper-badge.html

```diff
@@ -19,6 +20,13 @@
 the anchor element specified in the `for` attribute, or, if that doesn't exist,
 centered to the parent node containing it.
 
+Badges can also contain an icon by adding the `icon` attribute and setting
+it to the id of the desired icon. Please note that you should still set the
+`label` attribute in order to keep the element accessible.
+
+`paper-badge` includes a default icon set. See [iron-iconset](../iron-iconset)
```

Comment:
Ah sorry I think the docs here were unclear. You can definitely use any icon in any icon set with badges, so long as you import the icon set. `iron-icons.html` is imported within `paper-badge` for convenience. I'll update the docs and add a non-standard icon usage to the demo page.